Web/fluid_properties/Validation/NelsonValidation.py

```
@@ -1,40 +1,45 @@
+            h = HAPropsSI('H', 'T', tdb_ + 273.13, 'Twb', twb_ + 273.15, 'P', P) / 1000.0
+            tdp = HAPropsSI('Tdp', 'T', tdb_ + 273.13, 'Twb', twb_ + 273.15, 'P', P) - 273.15
+            W = HAPropsSI('W', 'T', tdb_ + 273.13, 'Twb', twb_ + 273.15, 'P', P)
+            R = HAPropsSI('R', 'T', tdb_ + 273.13, 'Twb', twb_ + 273.15, 'P', P) * 100
+            v = HAPropsSI('V', 'T', tdb_ + 273.13, 'Twb', twb_ + 273.15, 'P', P)
+        except ValueError:
+            # Some near-saturation points fall outside CoolProp's RH validity range
+            # (the header notes one such omission); skip rather than abort the table.
+            print("  (Tdb={Tdb:.0f}C, Twb={Twb:.0f}C omitted: outside validity range)".format(Tdb=tdb_, Twb=twb_))
+            continue
+        s = 0
+        print("{Tdb:10.2f}{Twb:10.2f}{Tdp:10.2f}{R:10.1f}{W:10.5f}{h:10.2f}{v:10.3f}".format(W=W, Twb=twb_, Tdp=tdp, Tdb=tdb_, v=v, h=h, s=s, R=R))
```

Ruff (0.15.15)

[error] `.format` call has unused named argument(s): s

Remove extra named arguments: s

(F522)